Library/Homebrew/utils.rb

```diff
@@ -124,20 +114,20 @@ def odie(error)
 def pretty_installed(f)
   if !$stdout.tty?
     "#{f}"
-  elsif ENV["HOMEBREW_NO_EMOJI"]
+  elsif !Emoji.enabled?
```

Comment:
To be honest, I think minimal diffs has already fallen on its sword when it comes to refactoring as major as this 🙈.